### DISCUSSION TITLE
Button without label

### DISCUSSION THREAD

**Discussion by ghost** [formkit/formkit#1081] [General]
👋
can't we use "Button" without label. when we need button picto for example without label.
you're make button required label or you're pass default label when it empty.
is there a solution for this ??

**Comment by justin-schroeder** [comment]
What makes you say the label is required?

**Reply by ghost** [reply]
@justin-schroeder 


<FormKit
  type="button"
  label=""
  help="You can use the label prop."
  prefix-icon="check"
/>

<FormKit
  type="button"  
  help="You can use the label prop."
  prefix-icon="check"
/>

the result of this components should not label appeared, but we get a default label "submit".
what if we don't need label, we just want buton picto without label.
did you get what i mean ?

**Reply by justin-schroeder** [reply]
Just put it in the default slot:

```
<FormKit type=“button”>
  <img src=“pic.png”>
</FormKit>
```

**Reply by ghost** [reply]
@justin-schroeder
I think if we can use it like this is better. we still work with formkit-icons without import another library of icons or image !
`<FormKit type='button' prefix-icon="check"/>`
what do you think ?

**Reply by fenilli** [reply]
You can also pass an empty space at label or use sectionsSchema to modify it.
```html
<FormKit type="button" label=" " />

<FormKit type="button" :sections-schema="{ label: { $el: null, children: [] } }" />
```

**Reply by ghost** [reply]
@fenilli @justin-schroeder  get the same result. the css not applicated.
![same-result-1](https://github.com/formkit/formkit/assets/99481334/418e79d4-33e0-4167-a27d-f2d5eb4dc545)
![same-result](https://github.com/formkit/formkit/assets/99481334/bec28caf-fdde-4910-b1c5-bbe793924f57)

**Reply by fenilli** [reply]
Yeh a button doesn't have the label section, its input but that comes with some problems... so the best way would be exporting the button and removing label there.
